components/blocks/expertBlock.tsx

```diff
@@ -0,0 +1,115 @@
+import Image from "next/image";
+import { type Template } from "tinacms";
+import { CustomLink } from "../customLink";
+
+export const ExpertBlock = (props) => {
```

Comment:
the props is a complex type - I think it would be best to define the type

Reply:
done